database/sqlite/build_list.go

```diff
@@ -171,3 +172,46 @@ func (c *client) GetRepoBuildList(r *library.Repo, filters map[string]string, pa
 
 	return builds, count, err
 }
+
+// GetRepoBuildListByEvent gets a list of all builds by org name and event type from the database.
+//
+// nolint: lll // ignore long line length due to variable names
+func (c *client) GetRepoBuildListByEvent(r *library.Repo, event string, page, perPage int) ([]*library.Build, int64, error) {
```

Comment:
I don't think we want to add this function as it was removed with:

https://github.com/go-vela/server/pull/443